package.json

```diff
@@ -87,7 +90,7 @@
     "@gorhom/bottom-sheet": "^4.1.5",
     "@pagopa/io-pagopa-commons": "^3.1.0",
     "@pagopa/react-native-cie": "^1.1.4",
-    "@pagopa/ts-commons": "^10.5.0",
+    "@pagopa/ts-commons": "^10.10.0",
```

Comment:
@francescopersico here the package updated

Reply:
We don't need to update the version here.
We are using **^**10.5.0 and the yarn.lock has already been updated to take the 10.10.0 version.